internal/pkg/processor/config/config.go

```diff
@@ -51,6 +57,15 @@ func LoadConfig(configPath string) (cfg *Config, err error) {
 		return nil, kbc.UserErrorf("invalid configuration: %s", processJSONError(err))
 	}
 
+	switch dataTypeSupport {
+	case "hint":
+		conf.Parameters.NewManifestMode = slicerConfig.NewManifestModeColumns
+	case "authoritative":
+		conf.Parameters.NewManifestMode = slicerConfig.NewManifestModeSchema
+	default:
+		return nil, kbc.UserErrorf(`undexpected data type support: "%s"`, dataTypeSupport)
```

Comment:
This doesn't match specification, ENV can be undefined (or I did miss something):

![image](https://github.com/keboola/processor-split-table/assets/19371734/9c294663-d56b-416c-b8b8-46f7e99e09ee)

Slack: https://keboolaglobal.slack.com/archives/C055HSMKX51/p1712570028466149?thread_ts=1712314989.537509&cid=C055HSMKX51

Reply:
:+1: